### DISCUSSION TITLE
Just a doubt.

### DISCUSSION THREAD

**Discussion by kajuburfi** [NNBnh/dotfiles#10] [Q&A] [answered]
Hey NNB, I have a doubt with rust lang. Hoping you could help.

Suppose I have a directory files in which there are some files named file1, file2, file3 and so on.
Is it possible to read any one file (stored in a variable) in rust?

Eg:
```rs
use std::fs;

fn main() {
    let file_name = ..;//Some code for choosing a file
    let file = fs::read_to_string("./files/{}", file_name); //Is it possible to do so?
    println!("{}", file);
}
```

Sorry, element had some issues.. So sent here.

**Comment by NNBnh** [comment, marked as answer]
Rust is on my _languages-to-learn_ list for quite while now but I never have time to focus on learning it...

So sorry... I don't have enough experience with Rust to help you much on this one.

> Suppose I have a directory files in which there are some files named file1, file2, file3 and so on.
> Is it possible to read any one file (stored in a variable) in rust?

I remember some similar thing like this with C++ so I'm pretty sure it's possible to do the same thing with any low-level programing languages include Rust. Keep try and error until it work, whisk you luck!

> Sorry, element had some issues.. So sent here.

Actually, you post this at the very right place! I open this discussion on my dotfiles repo just so people can reach out to me ^^.
